spec/requests/users_spec.rb

```diff
@@ -1,7 +1,23 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] This is for `User` and not post. 

👇🏾 
`RSpec.describe "Users", type: :request do`

Reply:
Thank you so much! @bobb-Rob